src/main/resources/application.yml

```
@@ -0,0 +1,40 @@
+
+--- #prod
+spring.config.activate.on-profile: prod
+server.port: ${SERVER_PORT:80} # default http
+spring:
+  datasource:
+    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_DATABASE_NAME}
+    username: ${DB_USERNAME}
+    password: ${DB_PASSWORD}
+  jpa:
+    hibernate:
+      ddl-auto: create-drop  # validate
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)